great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -3472,34 +3472,29 @@ def test_yaml_config(
 
         test_yaml_config is mainly intended for use within notebooks and tests.
 
-        Parameters
-        ----------
-        yaml_config : str
-            A string containing the yaml config to be tested
-
-        name: str
-            (Optional) A string containing the name of the component to instantiate
-
-        pretty_print : bool
-            Determines whether to print human-readable output
-
-        return_mode : str
-            Determines what type of object test_yaml_config will return
-            Valid modes are "instantiated_class" and "report_object"
-
-        shorten_tracebacks : bool
-            If true, catch any errors during instantiation and print only the
-            last element of the traceback stack. This can be helpful for
-            rapid iteration on configs in a notebook, because it can remove
-            the need to scroll up and down a lot.
-
-        Returns
-        -------
-        The instantiated component (e.g. a Datasource)
-        OR
-        a json object containing metadata from the component's self_check method
-
-        The returned object is determined by return_mode.
+        --Public API Whitelisted--
+
+        --Relevant Documentation--
+        /docs/terms/data_context
```

Comment:
If you include the full URL: https://docs.greatexpectations.io/docs/terms/data_context, then in most IDEs developers can click the link and jump straight to the docs from VS Code or PyCharm.

Reply:
Update has been made.